tfx/components/base/base_executor.py

```diff
@@ -160,3 +160,116 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
          output_dict: Dict[Text, List[types.Artifact]],
          exec_properties: Dict[Text, Any]) -> None:
     pass
+
+
+class BeamExecutor(BaseExecutor):
```

Comment:
Can you rename this `FuseableBeamExecutor`?

CC: @yifanmai who may have an opinion on this name (should we call this `ExperimentalFuseableBeamExecutor`?)

Reply:
`FuseableBeamExecutor` and `ExperimentalFuseableBeamExecutor` both sound good to me.

Reply:
Done, I changed it to `FuseableBeamExecutor`